src/behaviors/hv-set-value/index.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import type {
+  DOMString,
+  Document,
+  Element,
+  HvComponentOnUpdate,
+  HvGetRoot,
+  HvUpdateRoot,
+} from 'hyperview/src/types';
+import { later, shallowCloneToRoot } from 'hyperview/src/services';
+import {
+  setIndicatorsAfterLoad,
+  setIndicatorsBeforeLoad,
+} from 'hyperview/src/services/behaviors';
+
+export default {
+  action: 'set-value',
+  callback: (
+    element: Element,
+    onUpdate: HvComponentOnUpdate,
+    getRoot: HvGetRoot,
+    updateRoot: HvUpdateRoot,
+  ) => {
+    const targetId: ?DOMString = element.getAttribute('target');
+    if (!targetId) {
+      return;
+    }
+
+    const newValue: string = element.getAttribute('value') || '';
+
+    const delayAttr: string = element.getAttribute('delay') || '0';
+    const parsedDelay: number = parseInt(delayAttr, 10);
+    const delay: number = isNaN(parsedDelay) ? 0 : parsedDelay;
+
+    const showIndicatorIds: Array<string> = (
+      element.getAttribute('show-during-load') || ''
+    ).split(' ');
```

Comment:
There are a couple of usage of the space `' '` separator. Shall we make this and use as a constant say `EMPTY_SPACE`?